.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -28,6 +28,11 @@ on:
         required: false
         type: boolean
         default: false
+      cppwrapper:
+        description: Run inductor_cpp_wrapper for inference?
```

Comment:
It was for inference only, but now cpp-wrapper works for training too.

Reply:
@desertfire Can `aot_inductor` also work for training? I can update both while I'm at it!

Reply:
No, not for aot_inductor

Reply:
You can also update https://github.com/pytorch/pytorch/blob/b8f383107eebd9495a0f132d58a970e178e15930/.ci/pytorch/test.sh#L537 at the same time.